packages/@aws-cdk/aws-rds/test/integ.instance.lit.ts

```diff
@@ -49,7 +49,7 @@ class DatabaseInstanceStack extends cdk.Stack {
       instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE3, ec2.InstanceSize.MEDIUM),
       multiAz: true,
       storageType: rds.StorageType.IO1,
-      credentials: rds.Credentials.fromUsername('syscdk'),
+      credentials: rds.Credentials.fromFixedUsername('syscdk'),
```

Comment:
Can you add a new test instead, and leave the old one unchanged? This is such a risky change, we need all the help we can get here...